velox/docs/functions/window.rst

```diff
@@ -78,6 +78,16 @@ The query ranks orders for each clerk by price:
 Ranking functions
 =================
 
+.. function:: dense_rank() -> bigint
+
+Returns the rank of a value in a group of values. This is similar to rank(), except that tie values do
```

Comment:
Nit -- spelling "the"

Reply:
I have added ``the`` before ``tie values`` since I am not sure if ``the`` was misspelled anywhere, could you please elaborate what needs to be changed here?